### PR TITLE
Validation test ui (CORS changed)

### PR DESCRIPTION
**Pull request by adrianjost**
I have created a simple website to verify `.mergify.yml` configs. It basicly does the same thing, as your CURL Command, but CURL isn't nativly available on Windows and for beginner developers it also may be hard to use. My Website simply offers an input field where you can enter your config and then does a fetch to your domain to verify the config.

The Problem is that all modern browsers are blocking those request. So you need to explicitly allow requests from external domains. I have enabled those CORS Headers in your project for the routes `/validate` ~and `/convert`~ (**Edit:** I saw your pull request removing this feature completly).

I hope you like my work and will be happy to merge it. Otherwise, don't hestiate to close this pull, I've had some fun during development and that's what counting.

Demo: https://mergify.adrianjost.dev/
Repository: https://github.com/adrianjost/mergify-verify

[image](https://mergify.adrianjost.dev)

### CODE REVIEW

**Comment by jd** [comment]
That looks good to me. Is there any chance we could merge your app/form in the documentation directly?

**Comment by adrianjost** [comment]
that should be possible. Yes! It's realy just a super simple form with a fetch request. 
I can try to port it in the documentation directly if you wan't me to do so.

**Comment by jd** [comment]
Yeah that'd be great. That does not prevent this to be merged for now — though you'll need to rebase and resolve the conflict.

**Comment by adrianjost** [comment]
@jd I resolved the conflict, but I have never rebased before. Could you explain me, how to do it in this case?

**Comment by jd** [comment]
@adrianjost just type `git rebase origin/master` in your branch and then `git push -f` your branch back. That should be ok.

**Comment by adrianjost** [comment]
@jd thanks, it' done (hopefully correctly)

File: .env [hunk 1]
```diff
@@ -1 +1,8 @@
 COMPOSE_PROJECT_NAME=mergify
+MERGIFYENGINE_INTEGRATION_ID=11221
```
Comment (jd) [inline .env]: ?

File: README.rst [hunk 2]
```diff
@@ -10,3 +10,11 @@ mergify-engine
...
 
 Stop merging your pull requests manually, more info on https://mergify.io
+
+Development Setup
+=================
+
+pip install .
+set fake.env variables (Windows: set VARIABLE=XXX)
+SET FLASK_APP=mergify_engine/web.py
+python -m flask run
```
Comment (jd) [inline README.rst]: I'd prefer we focus on the goal of the PR, not sure those instructions are actually the "state of the art" :)

File: mergify_engine/web.py [hunk 3]
```diff
@@ -66,7 +68,24 @@ def badge(owner, repo):
                                      "badge-%s-%s.png" % (mode, style))
 
 
+@app.route("/convert", methods=["POST"])
```
Comment (jd) [inline mergify_engine/web.py]: uh?

**Comment by adrianjost** [comment]
Sorry @dj, seems like I screwed up the git rebase. I will revert all this tomorrow! Sorry again.

**Comment by adrianjost** [comment]
I hope it's better now? I accidently rebased to my forks master branch instead of yours.

**Pull request merged**